### PR TITLE
Add CI tool

### PR DESCRIPTION
**Pull request by at-kietvo**
Init config file.

### CODE REVIEW

**Comment by at-kietvo** [comment]
@at-tienhoang : I have finished config Circle CI. Let check my pull request. Thanks you guy.

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    #- ( sleep 5 && while [ 1 ]; do sleep 1; echo y; done ) | android update sdk --no-ui --all --filter "tools,android-25,build-tools-25.0.3,platform-tools,extra-google-m2repository,extra-android-m2repository,extra-android-support,extra-google-google_play_services"
+    - echo y | android update sdk --no-ui --all --filter "tool,extra-android-m2repository,extra-android-support,extra-google-google_play_services,extra-google-m2repository"
+    - echo y | $ANDROID_HOME/tools/bin/sdkmanager "extras;m2repository;com;android;support;constraint;constraint-layout-solver;1.0.2"
+
+  cache_directories:
+    - ~/.android
+test:
+  override:
+    - scripts/saddler.sh
+  post:
```
Comment (at-tienhoang) [inline circle.yml]: We dont need this function, please remove it

File: scripts/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```
Comment (at-tienhoang) [inline scripts/deploy.sh]: We don't need this file, please remove it

**Comment by at-kietvo** [comment]
@at-tienhoang : Plz check again. Thanks a lot.

**Pull request merged**